spec/acceptance/config_values_spec.rb

```diff
@@ -1,160 +1,196 @@
 require 'spec_helper_acceptance'
 
-describe 'Chocolatey Config' do
-  context 'MODULES-3035 - Add New Config Item' do
-    before(:all) do
-      backup_config
+describe 'chocolateyconfig' do
+  context 'create chocolateyconfig resource' do
+    include_context 'backup and reset config'
```

Comment:
is this something that is required for each test file or even the whole suite in general?

Reply:
I have loose feelings about this, insofar as I naively tried to run the scheduled_task tests on linux and they failed instead of skipping. I think, unless suite-wide marks each test as skipped (vs just hiding them), we should do before each blocks to skip unless windows.